odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -580,6 +580,16 @@ qualified name of action followed by parentheses containing the binding paramete
 all overloads of an [Action](#Action)| 
 qualified name of action| 
 <pre>`MySchema.MyAction`</pre>
+bound [Action](#Action) overload via container| 
+Navigation Property via container or Property via container, followed by bound Action overload| 
+<pre>`MySchema.MyEntityContainer/MyEntitySet` 
+<br>` /MyNavigationProperty` 
```

Comment:
What about paths without a navigation property, to differentiate actions on multiple entity sets sharing the same entity type?